assignments/array-methods.js

```diff
@@ -56,28 +56,65 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach(runner => {
+    fullName.push(runner.first_name + " " + runner.last_name);
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+allCaps = runners.map(runner => {
+    return runner.first_name.toUpperCase();
+})
 console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
 let largeShirts = [];
+largeShirts = runners.filter(runner => {
+    return runner.shirt_size === "L" ? true : false;
```

Comment:
this is a little redundant because an if statement will automatically return a true or false, 
```suggestion
    return runner.shirt_size === "L";
```
if you think of an if statement, the value in the parentheses after the IF keyword is always either true or false.